README.md

```diff
@@ -213,11 +213,11 @@ Example:
 

```

Comment:
- [ ] `README.md` file
```
## 🚀 Live Demo <a name="live-demo"></a>

> Add a link to your deployed project.

- [Live Demo Link](https://yourdeployedapplicationlink.com)
```
Kindly make sure you add a working link on your Live Demo section as the current one displays a blank page, this will make your README more professional. If you are not able to deploy your app online, I suggest you leave ther a message similar to these: `Available soon`, `N/A`, `Not available`,...